test.sh

```diff
@@ -66,44 +60,23 @@ function setup_osbs() {
 
   # RPM install basic dependencies
   $RUN $PKG $ENABLE_REPO install -y "${PKG_EXTRA[@]}"
-  [[ ${PYTHON_VERSION} == '3' ]] && WITH_PY3=1 || WITH_PY3=0
   # RPM install build dependencies for atomic-reactor
-  $RUN "${BUILDDEP[@]}" --define "with_python3 ${WITH_PY3}" -y atomic-reactor.spec
-  if [[ $OS == "centos" ]]; then
-    # RPM install dependencies for unit tests, as check is disabled for rhel
-    $RUN yum install -y python-flexmock python-six \
-                        python-backports-lzma \
-                        python-backports-ssl_match_hostname \
-                        python-responses \
-                        PyYAML \
-                        python-requests python-requests-kerberos # OSBS dependencies
-  else
+  $RUN "${BUILDDEP[@]}" --define "with_python3 1" -y atomic-reactor.spec
```

Comment:
`with_python3`  IMO this should be removed in commit where py3 support is removed from scpecfile